blackberggroup/VA-National-Centers
# feat: all centers page update

## What type of PR is this? (check all applicable)


## Description

## Related Tickets & Documents
[- Monday Task #](https://blackberg-group.monday.com/boards/1931719392/pulses/7465174603)

## QA Instructions, Screenshots, Recordings

Verify the hero CTA Button is removed 

Change "Collaborations and Project Spotlight" to "Collaboration and Project Spotlights"

https://dev-innovation.bgsandbox.com/ecosystem/views/national-centers/ideas/
https://dev-innovation.bgsandbox.com/ecosystem/views/national-centers/drhci/
https://dev-innovation.bgsandbox.com/ecosystem/views/national-centers/necie/
https://dev-innovation.bgsandbox.com/ecosystem/views/national-centers/ncchi/

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _includes/ideas/1_masthead.html
 - _includes/ideas/3_news.html
 - _site/ideas/index.html
